packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -29,6 +29,10 @@ const int kOverlineMask = 0x2;
 /// The mask constant for a line through or strike text decoration.
 const int kLineThroughMask = 0x4;
 
+/// This is Signature will report error when render an
+/// image
+typedef VGIErrorListener = void Function(Object error, StackTrace? stackTracr);
```

Comment:
Please spell this as `VectorGraphicsErrorListener`, avoid abbreviations in Dart code.